docs/cloud-native-security/cspm-get-started-azure.asciidoc

```diff
@@ -95,27 +104,66 @@ Before using this method, you must have set up a https://learn.microsoft.com/en-
 . Go to your Azure subscription list and select the subscription you want to monitor with CSPM.
 . Go to **Access control (IAM)** and select **Add Role Assignment**. 
 . Select the `Reader` function role, assign access to **User, group, or service principal**, and select your new app.
+. Return to the **Add CSPM** page in {kib}. 
+. Under **Configure integration**, select **Azure**. Under **Setup access**, select **Manual**.
+. Under **Preferred manual method**, select **Service principal with Client Secret**.
+. Fill in the values you copied earlier for **Client ID**, **Tenant ID**, and **Client Secret**.
+. Under **Where to add this integration**, select **New hosts**.
+. Click **Save and continue**, then follow the instructions to install {agent} on your selected host.
 
-On the VM where you plan to install {agent}, create the file `/etc/sysconfig/elastic-agent`. Paste the following content into the new file, substituting the values you copied for the placeholder values:
+Wait for the confirmation that {kib} received data from your new integration. Then you can click **View Assets** to see your data.
+
+[discrete]
+[[cspm-azure-client-certificate]]
+=== Option 3: Service principal with client certificate
 
+Before using this method, you must have set up a https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in[Microsoft Entra application and service principal that can access resources]. 
+
+. Go to the **Registered apps** section of https://ms.portal.azure.com/#view/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/~/RegisteredApps[Microsoft Entra ID].
```

Comment:
Perhaps before this step we could add the same steps as the `client secret` options regarding the role assignment? 

The steps are:
```
. Go to your Azure subscription list and select the subscription you want to monitor with CSPM.
. Go to **Access control (IAM)** and select **Add Role Assignment**. 
. Select the `Reader` function role, assign access to **User, group, or service principal**, and select your new app.
```